cg/models/observations/input_files.py

```diff
@@ -26,3 +26,10 @@ class BalsamicObservationsInputFiles(ObservationsInputFiles):
 
     snv_germline_vcf_path: FilePath
     sv_germline_vcf_path: FilePath | None = None
+
+
+class RarediseaseObservationsInputFiles(ObservationsInputFiles):
+    """Model for validating raredisease Loqusdb input files."""
```

Comment:
I definitely have OCD with Raredisease lower and uppercase 😆

```suggestion
    """Model for validating RAREDISEASE Loqusdb input files."""
```